mama/c_cpp/src/cpp/MamaDQPublisherManager.cpp

```diff
@@ -316,6 +317,11 @@ void MamaDQPublisherManager::destroyPublisher (const char *symbol)
 	mImpl->destroyPublisher (symbol);
 }
 
+MamaDQPublisher* MamaDQPublisherManager::getPublisher (const char* symbol)
+{
+    return mImpl->getPublisher (symbol);
```

Comment:
Thanks.
I get about mImpl, that's completely fair (I understand that this is an opaque pointer), but the class implementation has this method:
```void MamaDQPublisherManager::destroy (void)
{
    mImpl->destroy ();
    delete mImpl;
}```
that makes that pointer invalid? NULL?